index.md

```diff
@@ -48,6 +48,361 @@ def is_even(num):
+
+```python
+def info(age):
+    """
+    Return the info about my age.
+
+    @type age: int
+    @rtype: str
+    """
+    return "My age is" + age    # Error on this line
+                                # corrected version: "My age is" + str(age)
+info(2)
+```
+
+### E1131: unsupported-binary-operation {#E1131}
+
+For your work in python, we expect you to use two operands which support
+binary arithmetic operation. For example, a dictionary could not be added to
+a list. Make sure that you are adding objects which support binary arithmetic
+operation. This error may be a little similar to E1130.
+
+```python
+def my_fuc(t, n):
+    """Add <n> to <t>."""
+    if type(t) is tuple:
+        return t + (n, )
+    else:
+        return t + [n]  # Error on this line # thinks it is tuple
+
+my_fuc(1, 2)
+```
+
+### E1135: unsupported-membership-test {#E1135}
+
+For your work in python, we expect you to use an instance in membership test
+expression which implement membership protocol(__contains__/__iter__/__getitem__).
+This error may be similar to E1101. For example, if you want to check if a
+specific instance is inside another object, please make sure the object has this
+kind of membership.
+
+```python
+lst = 1132424
+if 'a' in lst:
+   print("unsupported membership test")  #Error on this line
+```
+
+### E1136: unsubscriptable-object {#E1136}
+
+For your work in python, we expect you to subscript value which support
+subscription (i.e. defined __getitem__ method). In the following example, b[0]
+could not work on 5, which is the third element of a(the list). As a result, we
+should not subscript value when we could not determine that all elements inside
+an object could be reached using the same method.
+
+
+```python
+a = [[1, 2], [5, 3], 5, [5, 6], [2, 2]]
+for b in a:
+    print(b[0])  # Error on this line
+```
+
+### E1303: format-needs-mapping {#E1303}
+
+For your work in python, we expect to see a format string that uses named
+conversion specifiers is used with an argument that is a mapping.
+
+```python
+gpl = "%(id)s : %(atr)s"
+
+objects = [{'id': 1, 'content': [{'atr': 'big', 'no': 2}]},
+           {'id': 2, 'content': [{'atr': 'small', 'no': 3}]}]
+
+for obj in objects:
+    for con in obj['content']:
+        print(gpl % (obj, con))    # Error on this line
+```
+
+
+### E1305: too-many-format-args {#E1305}
+
+For your work in python, we expect you to use one argument for a {}
+for format string. This error is similar to E1121, also opposite to E1306.
+In the following example, we give more values than {}. Remember to give a value
+to a {} when we are using format string.
+
+```python
+name = "Amy"
+age = "17"
+country = "England"
+city = "London"
+
+s = "{} who is {} lives in {}".format(name, age, country, city) # Error on this line
+```
+
+Corrected version
+
+```python
+name = "Amy"
+age = "17"
+country = "England"
+
+s = "{} who is {} lives in {}".format(name, age, country) # Error on this line
```

Comment:
You need to be careful with your corrected examples. The whole point is there is no longer and error on the specified line.